actions/update-artifacts-file/action.yaml

```diff
@@ -1,60 +1,74 @@
 name: "Update Artifacts File"
-description: "An action to update the artifacts file"
+description: "An action to update the artifacts file in a different repository"
+
+env:
+  TARGET_PATH: temp_helm_charts
```

Comment:
but you extended this by allowing the user input the repo. So, it defaults to 'helm-chart' but it does not have to be 'helm-charts'. The developer can put any repo so the path should be more generic, maybe: temp_repo